2024-RIKEN-AWS/JupyterNotebook/docker/Dockerfile.spawn

```diff
@@ -101,6 +102,10 @@ CMD ["flux", "start", "--test-size=4", "jupyter", "lab"]
 # This won't be available in K8s, but will be for a single container build
 COPY ./tutorial /home/jovyan/flux-tutorial-2024
 
+USER root
+RUN mv /home/jovyan/dlio_extensions /home/jovyan/flux-tutorial-2024
```

Comment:
Just want to double check the outcome of this - it puts dlio_extensions into flux-tturoial-2024? E.g.,

```
/home/jovyan/flux-tutorial-2024/dlio-extensions/
```
Any reason to not put the COPY command for the tutorial first and put the directory there in the same build step? It might get around needing to switch back and forth from the user again (but putting it all in the root block).

Reply:
You know... that makes a lot more sense. Let me change that quickly.

Reply:
Fixed

Reply:
That looks good! Have you run this notebook locally (with the updated build) and can confirm everything is working as expected?

Reply:
I haven't run since moving the COPY and mv, but everything was working before that. I tested it personally, and someone else in our lab here at UT tested it too.

Reply:
To be clear, when I say "working", I mean that everything builds, and the Flux side runs. I need to finish the new stuff for the DYAD notebook before I can test if that stuff works w.r.t running

Reply:
okay, let's wait then for that, and one final run through that everything is working. Then we can merge into the PR branch. My main desire is not to merge something that has not been fully tested to be working.

Reply:
Sounds good. Baring anything unexpected, I should have that done sometime tomorrow.